pootle/apps/contact/forms.py

```diff
@@ -56,6 +57,17 @@ def from_email(self):
     def recipient_list(self):
         return [settings.POOTLE_CONTACT_EMAIL]
 
+    def save(self, fail_silently=False):
+        """
+        Build and send the email message.
+
+        """
+        kwargs = self.get_message_dict()
+        kwargs["headers"] = {"reply-to": kwargs["from_email"]}
+        kwargs["from_email"] = settings.DEFAULT_FROM_EMAIL
```

Comment:
Works for me. But please address the other nits, specifically renaming to `Reply-To` to use exactly the same case that is expected.